convert2rhel/grub.py

```diff
@@ -171,6 +171,8 @@ def _get_device_number(device):
         raise BootloaderError("Unable to get information about the '%s' device" % device)
     # We are spliting the partition entry number, and we are just taking that
     # output as our desired partition number
+    if not output:
+        raise BootloaderError("The '%s' device has no PART_ENTRY_NUMBER" % device)
```

Comment:
Do you know if the error code in this case would be something other than `0`? If the `blkid` utility can give us a different error code in this case, I think we should use it instead of the output to raise this exception. 

If not, that's okay using output.